corehq/apps/accounting/management/commands/suppress_duplicate_invoices.py

```diff
@@ -70,6 +70,20 @@ def revert_invoice_payment(self, invoice, note, dry_run=False):
                 if not dry_run:
                     CreditLine.add_credit(amount=plan_credit, subscription=invoice.subscription,
                                         is_product=True, note=note)
+        else:
+            # Check if we generate duplicate product credit line
+            try:
+                CreditLine.objects.get(account=invoice.subscription.account,
+                                       subscription=invoice.subscription, isProduct=True, isActive=True)
+            except CreditLine.MultipleObjectsReturned:
+                print(f"Multiple plan credit lines found for subscription {invoice.subscription.id}.")
+                duplicate_cl = CreditLine.objects.filter(account=invoice.subscription.account,
+                                                         subscription=invoice.subscription, isProduct=True,
```

Comment:
just to be sure these are only the `CreditLine`s created by a previous run of the script, you should filter by `last_modified=datetime.datetime(2023, 8, 15)` or something like that :)

Reply:
Hi Biyeun, previous run of the script didn't create CreditLine. It's created when we generate invoice. I'll filter by `date_created=datetime.datetime(2023,08,01)`